CardIOCardholderNameTextFieldDelegate.m

```diff
@@ -0,0 +1,24 @@
+//
+//  CardIOCardholderNameTextFieldDelegate.m
+//  See the file "LICENSE.md" for the full license governing this code.
+//
+
+#import "CardIOCardholderNameTextFieldDelegate.h"
+
+@implementation CardIOCardholderNameTextFieldDelegate
+
+-(id) init {
+  if ((self = [super init])) {
+    // Globalization: alphanumeric, space, hyphen are all definitely okay;
+    // there's no compelling reason for us to get fussy here.
+    self.numbersOnly = NO;
+    self.maxLength = 175;  // PayPal REST APIs accept max of 175 chars for cardholder name
```

Comment:
I agree that it should be a constant, but, again, this is strictly following their style (of not using constants). I think I'll do what Dan did, and after this branch gets merged, make another PR for Card.io that cleans up their code and implements constants.